### DISCUSSION TITLE
Solving a permissions problem

### DISCUSSION THREAD

**Discussion by ExtremeNoob** [jlesage/docker-handbrake#281] [Q&A]
Hi!  I'm getting a message when loading the handbrake stack with portainer;
[nginx] nginx: [alert] could not open error log file: open() "/config/log/nginx/error.log" failed (13: Permission denied)
Being a noob, I installed and run omv as root.  Is there a troubleshooting method for this kind of situation?  Since the container doesn't start, I can't attach a console and inspect it from the inside.  I was trying to find the permissions for nginx and apply them to the folder in question, but so far, no luck.

Thanks!

**Comment by jlesage** [comment]
Can you share more details on how you created the container, what is its configuration ?
